android/java/AndroidManifest.xml

```diff
@@ -217,9 +217,16 @@
     android:exported="false"
     android:launchMode="singleTask"
     android:theme="@style/Theme.Chromium.Activity.Fullscreen" />
+
 <service android:name="com.brave.playlist.PlaylistVideoService"
     android:exported="false"/>
 
+    <service
+            android:name="com.brave.playlist.PlaylistDownloadService"
```

Comment:
This Service is using exoplayer lib to download files **https://github.com/brave-experiments/playlist-kotlin/blob/da5a721c0ea0c99ca2d2ad0015af988a8afce647/src/main/java/com/brave/playlist/PlaylistDownloadUtils.kt#L20.** 

We should use Chromium's network stack instead:
https://github.com/brave/security/issues/1202#issuecomment-1450252466